index.d.ts

```diff
@@ -21,13 +21,18 @@ declare namespace Eris {
   type AnyChannel = AnyGuildChannel | PrivateChannel;
   type AnyGuildChannel = GuildTextableChannel | AnyVoiceChannel | CategoryChannel | StoreChannel;
   type AnyVoiceChannel = VoiceChannel | StageChannel;
-  type ChannelTypes = Constants["ChannelTypes"][keyof Constants["ChannelTypes"]];
   type GuildTextableChannel = TextChannel | NewsChannel;
   type InviteChannel = InvitePartialChannel | Exclude<AnyGuildChannel, CategoryChannel>;
   type PossiblyUncachedTextable = Textable | Uncached;
   type PossiblyUncachedTextableChannel = TextableChannel | Uncached;
   type TextableChannel = (GuildTextable & GuildTextableChannel) | (Textable & PrivateChannel);
-  type VideoQualityMode = 1 | 2;
+  type VideoQualityMode = (Constants["VideoQualityMode"])[keyof Constants["VideoQualityMode"]];
```

Comment:
```suggestion
  type VideoQualityMode = Constants["VideoQualityMode"][keyof Constants["VideoQualityMode"]];
```

The () brackets are useless here